src/mods/common/analyzer/new/bpy.types.mod.rst

```diff
@@ -20,21 +20,53 @@
 
    .. method:: __getitem__(key)
 
-      :type key: int | str
+      :type key: int
```

Comment:
Why is `str` removed?

Reply:
The `bpy_prop_array` that I tested returned `AttributeError: bpy_prop_array[key]: invalid key, key must be an int` when I gave it a string key; if there are ones that do accept string keys I'll add it back in. 👍